app/3.0/docs/devguide/dom-template.md

```diff
@@ -4,169 +4,271 @@ title: DOM templating
 
 <!-- toc -->
 
-Many elements use a subtree of DOM elements to implement their features. DOM templating provides an easy way to create a DOM subtree for your element.
+Many elements use a subtree of DOM elements to implement their features. DOM templating provides 
+an easy way to create a DOM subtree for your element.
 
-By default, adding a DOM template to an element causes Polymer to create a shadow root for the element and clone the template into the shadow tree.
+By default, adding a DOM template to an element causes Polymer to create a shadow root for the 
+element and clone the template into the shadow tree.
 
 The DOM template is also processed to enable data binding and declarative event handlers.
 
 ## Specify a DOM template
 
-Polymer provides three basic ways to specify a DOM template:
+To specify a template, defining a `template` property on the constructor. When extending an 
+element, you can inherit or extend the superclass's template.
 
--   `<dom-module>` element. This allows you to specify the template entirely in markup,
-    which is most efficient for an element defined in an HTML import.
--   Provide a string template. This works well for elements defined entirely in JavaScript
-    (for example, in an ES6 module).
--   Retrieve or generate your own template element. This allows you to define how the template is
-    retrieved or constructed, and can be used to modify a superclass template.
+-   [Define a template property on the constructor](#templateobject).
+-   [Inherit a template from another Polymer element](#inherit).
 
-Polymer provides a default `template` getter that retrieves a template from the element's
-`<dom-module>`. You can override this getter to provide a string template or a generated template
-element.
+### Define a template property on the constructor {#templateobject}
 
-### Specify a template using dom-module
+To specif the element's template, define a `template` property on the element's 
+constructor. For example, you can create a static `template` getter. The template
+is retrieved and processed when the first instance of the element
+is upgraded.
 
-To specify an element's DOM template using a `<dom-module>`:
+The template getter must return an instance of `HTMLTemplateElement`. Use the `html` helper function
+to generate an `HTMLTemplateElement` instance from a JavaScript template literal.
```

Comment:
"Use the `html` helper function to generate..."

Do we need to say where to import it from?

"Import the `html` helper function from `polymer-element.js` and use it to generate..."

Reply:
It's in the sample, but I added a parenthetical here so as not to make the sentence any more complicated than it was... WDYT?

Reply:
seems fine to me